games-board/chessx/chessx-1.4.6.recipe

```diff
@@ -0,0 +1,64 @@
+SUMMARY="A free and open source chess database application"
+DESCRIPTION="With ChessX you can operate your collection of chess games \
+in many ways: browse, edit, add, organize, analyze, etc."
+HOMEPAGE="https://chessx.sourceforge.net/"
+COPYRIGHT="2005 Michal Rudolf"
```

Comment:
Sorry, I had noticed this earlier but forgot to leave a comment about it.
The program is still being updated, so I guess you should replace `2005` by `2005-2017` (or `2007-2017` if you are not sure about 2005) in `COPYRIGHT`.
Thanks!

Reply:
Now I know why I forgot to leave a comment about the `COPYRIGHT` text.
I wanted to see which authors we could add, from those listed in http://chessx.sourceforge.net/node/7 but there are no years associated to each author, so we would have to either dig this or leave an incomplete text.

Anyone has an idea of what we could put in `COPYRIGHT`? Thanks!

Reply:
After all, shall we mention all the authors (and key contributors) together?
```
COPYRIGHT="2005-2017 Michal Rudolf, Jens Nissen, Marius Roets, Sean \
Estabrooks, Rico Zenklusen, William Hoggarth, Ejner Borgbjerg, Bruno Rizzuti"
```
I don't think anyone would disagree with this, but if that happens I'm open to any suggestion :)
Thanks!

Reply:
It is not something we can decide ourselves. Either each contributor wants to keep his own copyright on his own work, or he reassigns it to someone else (when/where that is possible, for example in France it is not legally possible to do it). We should just follow on what upstream says, either at the header of their LICENCE file, or in their sourcecode headers. It is up to them to provide us with accurate information.

Reply:
OK. Thanks for your message.

I understand we should dig each header to find the information. That's OK for me, but I don't have the time to do that, and I wouldn't feel comfortable asking @tts2k to do that too.

Assuming nobody wants to search for the correct text, would we need to postpone the merge until we have that information?

Is there something we could put in `COPYRIGHT` for the merge before someone fixes it?

After all, what we have in `COPYRIGHT` is only informative. It's not supposed to be an authority.

What would you do here? Can't we make an exception and merge it with an incomplete (or incorrect) content?

Reply:
> It is up to them to provide us with accurate information.

So just ask upstream if they didn't bother to do so until now.

Reply:
Found it !
https://sourceforge.net/p/chessx/code/2415/tree//trunk/src/database/index.cpp#l3

Reply:
So as of version 1.4.6 we've these copyright holders:

- 2012-2016 Jens Nissen
- 2011 Nokia Corporation and/or it's subsidiary(-ies)
- 2005-2007, 2010 Michal Rudof
- 2010 P. Sereno
- 2014 Markus Kolb
- 1999-2004 Shane Hudson
- 1998-2005 Gilles Vollant
- 2005-2011 Sergey A. Tachenov
- 2005 Ejner Borgbjerg
- 2005-2007 Marius Roets
- 2005, 2006, 2007 Heinz R. Hopfgartner
- 2008 Aliaksandr Antonik

Extracted with this 15s craft

    find -type f -exec grep -i copyright '{}' + | grep -v .svn | grep -v i18n | awk -F: '!a[$1]++' | less